src/Illuminate/Routing/Concerns/HasCustomAttributes.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Illuminate\Routing\Concerns;
+
+use Illuminate\Support\Collection;
+use ReflectionAttribute;
+use ReflectionMethod;
+
+trait HasCustomAttributes
+{
+    public function callAction($method, $parameters)
+    {
+        $reflection = new ReflectionMethod($this, $method);
+
+        $attributes = $this->gteCustomAttributes($reflection);
```

Comment:
Looks like a typo in the method name, or was it intentional? :) 
```suggestion
        $attributes = $this->getCustomAttributes($reflection);
```